server/lib/middlewares/getUserAccessLevel.js

```diff
@@ -0,0 +1,26 @@
+const roleNames = {
+  admin: 'delegated-admin',
+  super: 'delegated-super-admin'
+};
+
+const checkRole = (data) => {

```

Comment:
Rewrite this whole thing:

 1. If user has the `Delegated Admin - User` role, then give him then set `req.role = 'User'`
 2. If the user has the `Delegated Admin - Administrator` role, then give him `req.role = 'Administrator'`
 3. Look for roles in these places:
        `user.roles`
        `user.app_metadata.roles`
        `user.app_metadata.authorization.roles`